File: app/src/main/java/com/dpis/module/home/DonateActivity.java
package com.dpis.module.home;

import com.dpis.module.LocalizedActivity;
import com.dpis.module.ui.compose.SupportActivityContent;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

public final class DonateActivity extends LocalizedActivity {
    public static Intent createIntent(Context context) {
        return new Intent(context, DonateActivity.class);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SupportActivityContent.installDonate(this);
    }
}

File: app/src/main/java/com/dpis/module/home/DonateActivity.kt
package com.dpis.module.home

import android.content.Context
import android.content.Intent
import android.os.Bundle
import com.dpis.module.home.presentation.installDonate
import com.dpis.module.settings.LocalizedActivity

class DonateActivity : LocalizedActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        installDonate()
    }

    companion object {
        @JvmStatic
        fun createIntent(context: Context): Intent =
            Intent(context, DonateActivity::class.java)
    }
}
